tests/test_matmul/net.c

```diff
@@ -145,6 +157,8 @@ static inline void multiply ()
     compare_tensors(result, C, IN_CH*OUT_CH);
     null_tensor(result, IN_CH*OUT_CH);
 
+    /*
+
     printf("\n-----> Profiling mm_unroll_8x1:\n");
```

Comment:
Is this under fix?

Reply:
yes, this has been fixed and needs to be like this